hnn_core/tests/test_drives.py

```diff
@@ -20,54 +20,31 @@ def test_external_drive_times():
     """Test the different external drives."""
 
     params = Params()
-    p_common, p_unique = create_pext(params,
-                                     params['tstop'])
```

Comment:
don't we need this for testing legacy mode?

Reply:
I don't think we need it here. The only point of keeping legacy mode is to maintain the exact final dipole response of the model as tested in `test_parallel_backends.py.

Reply:
Let's discuss this tomorrow ... I'm a little unclear what is the strategy here. I'm assuming the following tests help make sure that `legacy_mode` does what it's supposed to do. The dipole equality tests are a bit too crude ... they don't check if the "legacy mode drives" are indeed created.